contexts/food-safety-certification/v1.json

```diff
@@ -0,0 +1,38 @@
+{
+  "@context": {
+    "@protected": true,
+
+    "FoodSafetyCertificationCredential": {
+      "@id": "https://contexts.vcplayground.org/examples/food-safety-certification/vocab#FoodSafetyCertificationCredential"
+    },
```

Comment:
- Should this be using `/vocab`?  The other properties are not.
- Should this be using the `contexts` server at all?  Would be good to figure out what might work best so those paths could resolve to something like an index.html with docs in the future.  Defining that as something on the `contexts` server might be the wrong approach.  Where should a future dereferencable vocab live?

(My view is all related example resources should be in on spot, but this design is not going that direction at the moment.  Would be good to attempt to plan ahead enough to avoid breaking all the URLs later.)

Reply:
> All the examples where terms are only using `@id` could be set directly.

Done. [c2a2c50](https://github.com/credential-handler/vc-examples/pull/16/commits/c2a2c5082924d943e654bc0618843a90eaa7f595)

Reply:
> * Should this be using `/vocab`?  The other properties are not.

I've updated the others to use `/vocab` as well

> * Should this be using the `contexts` server at all?  Would be good to figure out what might work best so those paths could resolve to something like an index.html with docs in the future.  Defining that as something on the `contexts` server might be the wrong approach.  Where should a future dereferencable vocab live?

@davidlehn @gannan08 Given the limited time I've marked this as a future problem to be solved and filed an issue for now - https://github.com/credential-handler/vc-examples/issues/17.